module/zcommon/zpool_prop.c

```diff
@@ -116,6 +116,9 @@ zpool_prop_init(void)
 	zprop_register_number(ZPOOL_PROP_DEDUPRATIO, "dedupratio", 0,
 	    PROP_READONLY, ZFS_TYPE_POOL, "<1.00x or higher if deduped>",
 	    "DEDUP", B_FALSE, sfeatures);
+	zprop_register_number(ZPOOL_PROP_BRTRATIO, "brtratio", 0,
+	    PROP_READONLY, ZFS_TYPE_POOL, "<1.00x or higher if cloned>",
+	    "BRT", B_FALSE, sfeatures);
```

Comment:
"brtratio" is not very informative.  Why not "cloneratio", for example?

Reply:
I agree this is not the best name... I didn't want to use "clone" as it may be confused with 'zfs clone' and blockcloneratio seems too long. But I agree this should be changed. If people are ok with cloneratio, then it's fine by me.

Thank you Alex for looking at this!

Reply:
blockrefratio is better, but maybe a bit long